cegs_portal/search/views/v1/dna_features.py

```diff
@@ -215,6 +215,16 @@ def request_options(self, request):
     def get(self, request, options, data, chromo, start, end):
         features_paginator = Paginator(data, options["per_page"])
         feature_page = features_paginator.get_page(options["page"])
+        assembly_list = []
+        selected = False
+        all_assemblies = [GRCH38, GRCH37]
+
+        for ref_genome in all_assemblies:
+            if (options["assembly"] is None and not selected) or (options["assembly"] == ref_genome):
+                assembly_list.append((ref_genome, "selected", ref_genome))
+                selected = True
+            else:
+                assembly_list.append((ref_genome, "", ref_genome))
```

Comment:
Since `assembly_list` isn't used in the case that this view is called via htmx, this should be moved below the `if request.headers.get("HX-Request"):` block